nixos/tests/gnome3/flashback.nix

```diff
@@ -0,0 +1,98 @@
+# GNOME Flashback Terminal tests for various custom sessions.
```

Comment:
Terminal?

Reply:
Yeah, they pretty much test if a terminal shows up.
Btw, I've pretty much rewritten this entirely locally. Though there's issues with the python testing driver and other ports preventing me from re-PRing it.